hugo/layouts/papers/list-entry.html

```diff
@@ -3,16 +3,13 @@
 <p class="d-sm-flex align-items-stretch">
   <span class="d-block mr-2 text-nowrap list-button-row">
     {{- with $paper.url -}}
-    <a class="badge badge-primary align-middle mr-1" href="{{ . }}" data-toggle="tooltip" data-placement="top" title="Open PDF">
+    <a class="badge badge-primary align-middle mr-1" href="{{ . }}.pdf" data-toggle="tooltip" data-placement="top" title="Open PDF">
       pdf
```

Comment:
Link explicitly to the PDF, instead of the canonical URL, freeing up the canonical URL to be redirected to the paper page.

Reply:
Links can be external as well, and this would break them! I think it's the URL field that needs changing, instead of duct-taping the template.

Reply:
Fixed. I have now generated `pdf`, `bib`, `mods`, and `endf` fields in the YAML, with the full URLs, and those are used to generate the links.